apps/wallet/src/ui/app/pages/home/tokens/TokensDetails.tsx

```diff
@@ -24,15 +23,14 @@ type TokenDetailsProps = {
 
 function MyTokens() {
     const accountAddress = useActiveAddress();
-    const { data: balance, isLoading: loadingBalances } =
-        useGetAllBalances(accountAddress);
+    const { data: balance } = useGetAllBalances(accountAddress);
 
     const noSuiToken = !balance?.find(
         ({ coinType }) => coinType === SUI_TYPE_ARG
     );
 
     return (
-        <Loading loading={loadingBalances}>
+        <>
```

Comment:
I seems we need the loading state here. Until balances load we will show the no sui found right? Why do we have to remove it?

Reply:
See my comment below